### PR TITLE
Geography Results

### PR DESCRIPTION
**Pull request by sarahc7**
Added a results scene and changed name scheme to match files in group repo. Cleaned up my code and did some modularization.

### CODE REVIEW

**Review by gaurnett — APPROVED** [review]
LGTM

File: sdk/webhooks/ActionsOnGoogleFulfillment/geography/countries.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module.exports.countries = [
+    ['Nigeria', 'Abuja', '002'],
```
Comment (mithasgoogle) [inline sdk/webhooks/ActionsOnGoogleFulfillment/geography/countries.js:2]: What's 002 here?  Not understanding the third field of each entry.

**Review by mithasgoogle — APPROVED** [review]
Great work!  One question inline.

File: public/css/main.css [hunk 2]
```diff
@@ -67,6 +67,30 @@ path {
 
 #map {
     position: relative;
-    height: 100vh;
-    width: 100vw;
+}
```
Comment (gaurnett) [inline public/css/main.css:70]: Let's create a css file for geography and import it in index.html

File: sdk/webhooks/ActionsOnGoogleFulfillment/geography/countries.js (hunk 1 above)
Comment (sarahc7) [reply sdk/webhooks/ActionsOnGoogleFulfillment/geography/countries.js:2]: The 3rd field is the subcontinent region for each country. In the GeoCharts API, regions are 3-digit codes specified by the United Nations Statistics Division, and I'm specifying regions in my maps to focus in on the countries I'm asking about.

**Review by Arboghast — APPROVED** [review]
Consider moving the functional logic in scene.js to the GeographyMain class

**Pull request merged**